runtime/themes/black_and_white.toml

```diff
@@ -0,0 +1,140 @@
+"diagnostic.unnecessary"          = { modifiers = ["dim"] }
+"diagnostic.deprecated"           = { modifiers = ["crossed_out"] }
+
+"error"                           = { fg = "foreground" }
+"hint"                            = { fg = "foreground" }
+"info"                            = { fg = "foreground" }
+"warning"                         = { fg = "foreground" }
+
+"diff.delta"                      = { fg = "foreground" }
+"diff.minus"                      = { fg = "foreground" }
+"diff.plus"                       = { fg = "foreground" }
+
+"function"                        = { fg = "foreground" }
+"function.builtin"                = { fg = "foreground" }
+"function.call"                   = { fg = "foreground" }
+"function.macro"                  = { fg = "foreground" }
+"function.method"                 = { fg = "foreground" }
+
+"keyword"                         = { fg = "foreground" }
+"keyword.control.conditional"     = { fg = "foreground" }
+"keyword.control.exception"       = { fg = "foreground" }
+"keyword.control.import"          = { fg = "foreground" }
+"keyword.control.repeat"          = { fg = "foreground" }
+"keyword.directive"               = { fg = "foreground" }
+"keyword.function"                = { fg = "foreground" }
+"keyword.operator"                = { fg = "foreground" }
+"keyword.return"                  = { fg = "foreground" }
+"keyword.storage"                 = { fg = "foreground" }
+"keyword.storage.modifier"        = { fg = "foreground" }
+"keyword.storage.type"            = { fg = "foreground", modifiers = ["italic"] }
+
+"label"                           = { fg = "foreground" }
+
+"markup.bold"                     = { fg = "foreground", modifiers = ["bold"] }
+"markup.heading"                  = { fg = "foreground", modifiers = ["bold"] }
+"markup.italic"                   = { fg = "foreground", modifiers = ["italic"] }
+"markup.link.text"                = { fg = "foreground" }
+"markup.link.url"                 = { fg = "foreground" }
+"markup.list"                     = { fg = "foreground" }
+"markup.quote"                    = { fg = "foreground", modifiers = ["italic"] }
+"markup.raw"                      = { fg = "foreground" }
+"markup.strikethrough"            = { modifiers = ["crossed_out"] }
+
+"punctuation"                     = { fg = "foreground" }
+"punctuation.bracket"             = { fg = "foreground" }
+"punctuation.delimiter"           = { fg = "foreground" }
+"punctuation.special"             = { fg = "foreground" }
+
+"special"                         = { fg = "foreground" }
+
+"string"                          = { fg = "foreground" }
+"string.regexp"                   = { fg = "foreground" }
+"string.special"                  = { fg = "foreground" }
+"string.symbol"                   = { fg = "foreground" }
+
+"tag"                             = { fg = "foreground" }
+"tag.attribute"                   = { fg = "foreground" }
+"tag.delimiter"                   = { fg = "foreground" }
```

Comment:
Almost all of these definitions should be redundant since everything inherits from ui.text.

Reply:
Redundant declarations have been removed. Please note that `ui.popup` has to be manually [assigned](https://github.com/helix-editor/helix/pull/11872/files#diff-e5f14f4661338fe08d872f2e9d511c5f3a6b2ebf360f5b44d2b33c4e5714bcc7R37-R39). It's unclear if and what certain declarations are inherited by ui.text